app-arch/torrentzip/torrentzip-1.1.ebuild

```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2023 Gentoo Authors
```

Comment:
Sorry about the delay. Let's skip 1.1 then, and go straight for 1.2.

Reply:
The push 2 days ago should address all your suggestions, but the "changes requested" warning doesn't go away. You attached this comment to a line which didn't change.